mathesar/api/viewsets/functions.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework import viewsets
+from rest_framework.response import Response
+from db.functions.base import supported_db_functions
+from mathesar.api.serializers.functions import DbFunctionSerializer
+
+
+class DbFunctionViewSet(viewsets.ViewSet):
```

Comment:
Please write a test for this API to ensure that it works and has the expected format.